runtime/lua/vim/_defaults.lua

```diff
@@ -534,14 +534,48 @@ do
     end,
   })
 
+  local nvim_terminal_shell_integration_ns =
+    vim.api.nvim_create_namespace('nvim.terminal.shell_integration')
+  vim.api.nvim_create_autocmd('TermRequest', {
+    group = nvim_terminal_augroup,
+    desc = 'Mark shell prompts indicated by OSC 133 sequences for navigation',
+    callback = function(args)
+      if string.match(args.data.sequence, '^\027]133;A') then
+        local lnum = args.data.cursor[1] ---@type integer
+        vim.api.nvim_buf_set_extmark(args.buf, nvim_terminal_shell_integration_ns, lnum - 1, 0, {})
+      end
+    end,
+  })
+
+  ---@param ns integer
+  ---@param buf integer
+  ---@param direction 'forward'|'backward'
+  local function jump_to_prompt(ns, win, buf, direction)
```

Comment:
Fine to keep but just to mention - in places like vim.snippet direction is 1/-1

(Alternatively this could also mirror vim.diagnostic.jump and support a count?)

Reply:
Good call, updated to use `count` 1a694c9b2104